ReactApp/src/contexts/user-context.js

```diff
@@ -52,6 +54,7 @@ export const UserProvider = ({ children }) => {
     const fetchUsers = async () => {
       try {
         const response = await requestServer(serverPath.GET_ALL_USERS, httpsMethod.GET);
```

Comment:
Please add validation to this response object, even if partial.
pull and merge master branch, if you need help contact me